build.sbt

```diff
@@ -34,6 +34,8 @@ ThisBuild / libraryDependencies ++= {
   }
 }
 
+ThisBuild / Test / testOptions += Tests.Argument(TestFrameworks.JUnit, "-v")
```

Comment:
This causes the `JUnit` test runner to log the test start/complete notifications to stdout, so we get console output of the test suite status as it runs.